# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/alsa-firmware/hotfix-stdint.patch.musl
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- alsa-firmware-1.2.4/hdsploader/tobin.c.vanilla	2024-04-23 14:37:17.705061713 +0200
+++ alsa-firmware-1.2.4/hdsploader/tobin.c	2024-04-23 14:37:24.683062004 +0200
@@ -2,6 +2,7 @@
 #include <stdlib.h>
 #include <unistd.h>
 #include <endian.h>
+#include <stdint.h>
 
 #include "digiface_firmware.dat"
 #include "digiface_firmware_rev11.dat"
